utils/rpms/daos.spec

```diff
@@ -586,6 +590,9 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+*Wed Mar 13 2024 Lei Huang <lei.huang@intel.com> 2.5.100-17
```

Comment:
You need to update https://github.com/daos-stack/daos/blob/7fda6883af1136cf0e5a614f772cbd20c7017b7b/utils/rpms/daos.spec#L18 to `17` to reflect this.

Reply:
Thank you very much! I will fix this in next commit.

Reply:
Fixed in the new commit. Thank you!